src/ids-text/ids-text.d.ts

```diff
@@ -1,6 +1,9 @@
 // Ids is a JavaScript project, but we define TypeScript declarations so we can
 // confirm our code is type safe, and to support TypeScript users.
 
+/**
```

Comment:
```suggestion
/** A segment of text with standardized settings, theming and fonts*/
```

Been doing these as oneliners...

Reply:
wasn't aware that it was possible but since JSDoc/VSCode seem to pick it up then makes sense!